Gemfile

```diff
@@ -24,11 +24,11 @@ gem 'factory_bot_rails' # used in rake db:seed in production
 gem 'ffaker'
 gem 'execjs'
 # shivani - changed from jwt 0.1.13 to 1.2.0
-gem 'jwt', '~> 1.2.0'
+gem 'jwt', '~> 1.5.0'
```

Comment:
Not sure we need to pin to a specific version here

Reply:
This was the minimum version new `twilio-ruby` needed. Above this version `jwt` was upgrading it's dependent gem.